lib/oxide-vpc/tests/integration_tests.rs

```diff
@@ -2248,6 +2247,17 @@ fn verify_dhcpv6_essentials<'a>(
         client_id,
         reply.find_option(dhcpv6::options::Code::ClientId).unwrap()
     );
+
+    // Assert FQDN is correctly constructed.
```

Comment:
A high-level comment on integration test coverage. It looks like we have DHCP6 coverage but not DHCP4?

Reply:
AFAIK DHCPv4 doesn't have a single FQDN option like DHCPv6 but instead separate Hostname and Domain options. But yea some DHCPv4 coverage would be welcome.